src/bitmessageqt/__init__.py

```diff
@@ -4644,12 +4614,10 @@ def treeWidgetItemChanged(self, item, column):
         if column != 0:
             return
         # only account names of normal addresses (no chans/mailinglists)
-        if any(
-                [
-                    not isinstance(item, Ui_AddressWidget),
-                    not self.getCurrentTreeWidget(),
-                    self.getCurrentTreeWidget().currentItem() is None,
-                ]
+        if (
+                not isinstance(item, Ui_AddressWidget) or
+                not self.getCurrentTreeWidget() or
+                self.getCurrentTreeWidget().currentItem() is None
```

Comment:
My C background is causing me to code like this.

Reply:
Pylint likes the boolean operators at the end of the line, too.